Lib/idlelib/idle_test/test_config_key.py

```diff
@@ -22,12 +23,40 @@ def tearDownClass(cls):
         cls.root.destroy()
         del cls.root
 
-
     def test_init(self):
         dia = config_key.GetKeysDialog(
             self.root, 'test', '<<Test>>', ['<Key-F12>'], _utest=True)
         dia.Cancel()
 
```

Comment:
I was thinking of adding more tests and looks like I should have because of the bug.  Sorry that the PR wasn't complete.